### DISCUSSION TITLE
Thank you for your contributions in March 2024

### DISCUSSION THREAD

**Discussion by MisRob** [learningequality/kolibri#12053] [Community]
This month, we'd like to to once again express our heartfelt gratitude to all contributors, whether you're new or have been with us for months. 

In March, we've merged pull requests from @iskipu, @GarvitSinghal47, @Abhishekzod007, @thesujai, @EshaanAgg, @AymanHammadi, @poju3185, @himanshuc3, @kafukoM, and @Wck-iipi, and there's plenty of other great work in progress. We've seen many bugfixes that will significantly improve the experience for learners and coaches, whole new features, new test suites that will help with stability of our applications, automation scripts to boost our efficiency, security enhancements, wonderful progress in introducing design system components to Studio, and improvements of development tooling along with some maintenance refactoring.

Recently, the whole team was amazed to find out that in one of our upcoming releases, the amount of merged pull requests authored by volunteers outnumbers those authored by the core team :yellow_heart: 

As always, we'll acknowledge all of you in our next [Community Newsletter](https://learningequalitynewsletter.getresponsepages.com/).

My thanks goes also to all team members who support contributors by helping with running a development server, providing helpful feedback and submitting thorough reviews. I hope you all have fun together and learn from each other. I certainly do :slightly_smiling_face:

Lastly, I'd like to briefly mention that based on the survey we conducted several months ago, we're taking steps to enhance our engagement with the community of contributors. If you're interested in connecting with Learning Equality and other volunteers in new ways, keep an eye on this forum where we'll be making an announcement by the beginning of May.

Wishing you all a restful weekend
Michaela

**Comment by radinamatic** [comment]
Thank you  @iskipu, @GarvitSinghal47, @Abhishekzod007, @thesujai, @EshaanAgg, @AymanHammadi, @poju3185, @himanshuc3, @kafukoM, and @Wck-iipi, your contributions are invaluable! 🥇 💟

**Comment by rtibbles** [comment]
Thank you all for your contributions - they make a huge difference to the users of our software across the world!
